sidebars.js

```diff
@@ -65,199 +74,223 @@ const sidebars = {
+                { type: "doc", label: "Quality Metrics", id: "/articles/metrics" },
+                { type: "doc", label: "Status Pages", id: "testkube-pro/articles/status-pages" },
+              ],
+            },
+            {
+              type: "category",
+              label: "Triggers",
+              items: [
+                { type: "doc", label: "Test Triggers", id: "articles/test-triggers" },
+                // { type: "doc", label: "K8 Eventss", id: "" },
+                // { type: "doc", label: "Testkube Events", id: "" },
+              ],
+            },
+            { type: "doc", label: "Webhooks", id: "articles/webhooks" },
+            {
+              type: "category",
+              label: "Testkube CLI",
+              items: [
+                { type: "doc", label: "Install the Testkube CLI", id: "articles/install/cli" },
+                { type: "doc", label: "Install Testkube with the CLI", id: "/articles/install/install-with-cli" },
+                { type: "doc", label: "Managing CLI Context", id: "testkube-pro/articles/managing-cli-context" },
+                { type: "doc", label: "Docker CLI", id: "articles/testkube-cli-docker" },
+                { type: "doc", label: "OAuth for CLI", id: "articles/oauth-cli" },
+              ],
+            },
+            {
+              type: "category",
+              label: "Advanced Features",
+              items: [
+                { type: "doc", label: "Cached Test Results", id: "testkube-pro/articles/cached-results" },
+                { type: "doc", label: "Templates", id: "/articles/templates" },
+                { type: "doc", label: "CDEvents", id: "articles/cd-events" },
+                { type: "doc", label: "Generating Test CRDs", id: "articles/generate-test-crds" },
+              ],
+            },
+            // { type: "doc", label: "Deprecated Features", id: "" },
+          ]  
     },
     {
       type: "category",
-      label: "Test Types",
-      link: {
-        type: "generated-index",
-        description: "Supported Test Types / Executors within Testkube",
-      },
+      label: "Administration",
       items: [
-        "test-types/executor-artillery",
-        "test-types/executor-curl",
-        "test-types/executor-cypress",
-        "test-types/executor-ginkgo",
-        "test-types/executor-gradle",
-        "test-types/executor-jmeter",
-        "test-types/executor-k6",
-        "test-types/executor-kubepug",
-        "test-types/executor-kubent",
-        "test-types/executor-maven",
-        "test-types/executor-playwright",
-        "test-types/executor-postman",
-        "test-types/executor-pytest",
-        "test-types/executor-soapui",
-        "test-types/executor-tracetest",
-        "test-types/executor-zap",
-        "test-types/prebuilt-executor",
-        "test-types/container-executor",
-        "test-types/executor-distributed-jmeter",
```

Comment:
suggest to also create a "Legacy Tests" submenu that contains copies of all of these pages? (since these will be rewritten for workflows)